websites/G/GetPocket/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "RedBird16.com",
+    "id": "444872133504663552"
+  },
+  "service": "Pocket",
+  "description": {
+    "en": "Pocket"
```

Comment:
this should be Pocket description of website.

Reply:
Thank you for that.